tensorflow/compiler/xla/python/BUILD

```diff
@@ -82,6 +82,18 @@ py_test(
     ] + xla_py_test_deps(),
 )
 
+py_test(
```

Comment:
I will note that the existing tests for this functionality are in the JAX tree:
https://github.com/google/jax/blob/main/tests/tree_util_test.py

I might be tempted to add your new tests there instead. You have to do it as a separate PR; the way you make a change across both repositories is that you bump the client version number in xla_client.py in this repository (in this directory), and then gate your new test on that version (exposed in jax as `jax._src.lib.xla_extension_version`).